lib/core/src/text/buffer/fragment.rs

```diff
@@ -292,6 +296,14 @@ impl BufferFragments {
         }
     }
 
+    /// Mark as dirty all fragments with dirty assigned line.
```

Comment:
TBH, I completely do not understand this doc comment :P

Reply:
This is a method of BufferFragments. Fragments may be dirty (that's comes from their definition described in `BufferFragment` doc). Lines also may be dirty. Fragments may have line assigned. 

I assume that the reader will have knowledge of above facts. And I hope I don't have to explain what `dirty` means?